controllers/auth-controller.js

```diff
@@ -0,0 +1,18 @@
+var exports = module.exports = {}
+exports.signup = function(req, res) {
+  res.render("signup");
+};
+
+exports.signin = function(req, res) {
+  res.render("signin");
+};
+
+exports.dashboard = function(req, res) {
+  res.render("dashboard");
+};
+
+exports.logout = function(req, res) {
+  req.session.destroy(function(err) {
+    res.redirect("/");
+  });
+};
```

Comment:
it looks like all this is doing is rendering handlebars, couldn't we just redirect to the "user" handlebars and pass in the user id through the second argument of res.render?

Reply:
should the format var exports = module.exports = {} be formatted like we usually do for consistency?

module.exports = function () {
 }

?